fabric-object-builders-v0/src/main/java/net/fabricmc/fabric/api/block/BlockSettingsExtensions.java

```diff
@@ -38,50 +38,50 @@ public static void breakByTool(Settings settings, Tag<Item> tag, int miningLevel
 	}
 
 	public static void hardness(Settings settings, float hardness) {
-		((BlockSettingsHooks) settings).setHardness(hardness);
+		((BlockSettingsAccessor) settings).setHardness(hardness);
```

Comment:
This is might conflict with #530 for sure with all the Hooks -> Accessor changes, but it shouldn't be that much of a change when you rebase